src/components/form.jsx

```diff
@@ -0,0 +1,61 @@
+export default function Form({ handleSubmit, form, onChange }) {
+	return (
+		<>
+			<form onSubmit={handleSubmit} className="">
+				<label htmlFor="itemName">Item Name:</label>
+				<input
+					type="text"
+					id="itemName"
+					name="itemName"
+					value={form.itemName}
+					onChange={onChange}
+					required
+				/>
+				<fieldset>
```

Comment:
Not sure if this is just on my end, but on the Add Item page, my first click on any of the radio buttons or their labels selects the 'soon' option and I have to click a second time to get the option I want to be selected.
For example:
- when the page first loads, 'soon' is selected by default, I need to click on the radio button/label for 'Not Soon' twice before it will be selected
- if I already have 'Not Soon' selected, my first click on 'Kind of Soon' will cause the 'soon' option to be selected, then my second click on 'Kind of Soon' will cause 'Kind of Soon' to be selected.

Reply:
We were having that issue on our end as well. I will look into later to it to see what could've been the cause.

Reply:
I was looking into this and we might just need to remove the preventDefault in the clickHandler function and it should be fine.

Reply:
Ah! I got it. Changing the "checked" property to "defaultChecked" on the radio selection that I want set as the default fixed the issue.

Reply:
@mentalcaries @a-fink I've fixed this bug. Feel free to check it on your end to make sure it works properly.

Reply:
Awesome! Looks good on my end. Nice catch on the defaultChecked.

Reply:
@Amanda2900 getting close! With `defaultChecked`, the "soon" button is selected on the initial render.
If I submit an item as "kind of soon" or "not soon", that button stays selected.
If I leave it as is and submit another item, it `daysUntilNextPurchase` is submitted as `7`. Add a `console.log(form)` to the component and try it

Reply:
@mentalcaries I think I'm confused. I am logging the form at the end of the handle submit. It is registering the correct daysUntilNextPurchase that I select. The default is set to 7, which is the soon selection and also the form in the useState is set to 7 this way the form will never be submitted with that daysUntilNextPurchase blank. So we set it up that it will always pick 7 unless stated otherwise. And then when the form resets it automatically sets Soon back to the default selection. Are we not supposed to have a default set? As far as how we had planned to set this up it is working as intended on my end unless I'm missing an error.

Reply:
Nice work team! Looks great!